### ISSUE TITLE
Error Frame' object has no attribute 'to_jay'

### ISSUE THREAD

**Issue by Cheburusska** [h2oai/datatable#1716]
```
import datatable as dt
import math

DT = dt.Frame(A=range(5), B=[1.7, 3.4, 0, None, -math.inf], C = ['two','one','one','two','two'],
                 stypes={"A": dt.int64})
DT.to_jay("data.jay")
```

returns error

```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-76-f85faaa0a6ad> in <module>
      4 DT = dt.Frame(A=range(5), B=[1.7, 3.4, 0, None, -math.inf], C = ['two','one','one','two','two'],
      5                  stypes={"A": dt.int64})
----> 6 DT.to_jay("data.jay")

AttributeError: 'Frame' object has no attribute 'to_jay'
```

**Comment by jangorecki** [comment]
what datatable version are you using? `dt.__version__` if dev then also `dt.__git_revision__`

**Comment by Cheburusska** [comment]
@jangorecki latest version from pip '0.8.0'. Also you can check in documentation. It is missing there

**Comment by jangorecki** [comment]
https://github.com/h2oai/datatable/pull/1686 this was tagged for 0.9.0 milestone

**Comment by Cheburusska** [comment]
Ok, got it but it exists in documentation version 0.7.0 so this is pretty confusing.

**Comment by st-pasha** [comment]
`.to_jay()` function was added only recently, as @jangorecki rightfully notes. It is considered "latest dev", which will be eventually released as version 0.9.0. This function did not exist in 0.8.0 or 0.7.0.

On documentation website (readthedocs.io), this function appears in the [latest version](https://datatable.readthedocs.io/en/latest/api/frame.html#datatable.Frame.to_jay), but not in [version 0.7.0](https://datatable.readthedocs.io/en/v0.7.0/) as far as I can see, nor in [version 0.8.0](https://datatable.readthedocs.io/en/v0.8.0/) which doesn't even exist.

The issue of 0.8.0 docs not being there is, of course, a problem too. But it's not an easy thing to fix: apparently RTD has switched their backend python version from 3.5 to 3.7, and now it fails during documentation build process due to wheel version mismatch.

Hopefully this resolves this question, if not feel free to reopen.

**Comment by st-pasha** [comment]
(the documentation for 0.8.0 is now fixed too, so the URL above should work now)
